helm/aws-operator/templates/01-rbac.yaml

```diff
@@ -103,20 +103,20 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: aws-operator
+  name: aws-operator.v[[ .Version ]]
 subjects:
   - kind: ServiceAccount
-    name: aws-operator
+    name: aws-operator.v[[ .Version ]]
     namespace: {{ .Values.namespace }}
 roleRef:
   kind: ClusterRole
-  name: aws-operator
+  name: aws-operator.v[[ .Version ]]
   apiGroup: rbac.authorization.k8s.io
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: aws-operator-psp
+  name: aws-operator-psp.v[[ .Version ]]
```

Comment:
Why don't we make this a bit cooler with variables from the values file? This here is really not so cool.